main.py

```diff
@@ -98,7 +100,17 @@ async def get_forecasts(request: schemas.WeatherForecastRequest):
 async def get_stations():
     """ Return a list of fire weather stations.
     """
-    return schemas.WeatherStationsResponse.parse_file('data/weather_stations.json')
+    try:
+        use_wfwx = config.get('USE_WFWX') == 'True'
+        if use_wfwx:
+            LOGGER.info('Using WFWX to retrieve station list')
+            stations = await wildfire_one.get_stations()
+            return schemas.WeatherStationsResponse(weather_stations=stations)
+        LOGGER.info('Using pre-generated json to retrieve station list')
+        return schemas.WeatherStationsResponse.parse_file('data/weather_stations.json')
+    except Exception as exception:
+        LOGGER.critical(exception, exc_info=True)
+        raise
```

Comment:
Does this return http error to the client?

Reply:
Yes, it re-throws whatever the source exception was. In this case the client just gets an internal server error.